vkcs/monitoring/resource_cloud_monitoring.go

```diff
@@ -199,7 +199,7 @@ func (r *сloudMonitoringResource) Delete(ctx context.Context, req resource.Dele
 		return
 	}
 
-	err = monitoring.DeleteUser(monitoringClient, projectID, data.ServiceUserID.String()).Err
+	err = monitoring.DeleteUser(monitoringClient, projectID, data.ServiceUserID.ValueString()).Err
 	if err != nil {
```

Comment:
Here you should return silently if an error is 404.

Reply:
done